bot.go

```diff
@@ -56,6 +57,13 @@ func (b *Bot) Start(timeout time.Duration) {
 	b.poll(b.Messages, b.Queries, b.Callbacks, timeout)
 }
 
+// Stop method closes shutdown channel, which is used by listen
+// updates cycle. As soon as shutdown was closed poll cycle close
+// all messages channels
+func (b *Bot) Stop() {
+	close(b.shutdown)
```

Comment:
This will panic if `Stop()` is called more than once. At a minimum, we should mention this in the comment for this function. Better would be if consecutive calls of stop would be a no-op.

Reply:
Oh, ok, i have fixed it. But best way is make mutex lock for close operation to prevent closing from another goroutine. But in this case my fix is suffice too.